floater/generators.py

```diff
@@ -88,15 +103,24 @@ def get_rectmesh(self):
         RETURNS
         -------
         x : np.ndarray
-            2D array of float x coordinates
+            3D array of float x coordinates
```

Comment:
If the shape of the outputs is now different, I am worried that will break backwards compatibility. What would really be ideal is if we return a 2D array for 2D cases and a 3D array for 3D cases.

On the other hand, maybe this function is not meant to be called directly, but rather will just be called by `to_mitgcm_format`. I honestly don't know who is using this and how.